src/components/LogosSection/Achievements/index.js

```diff
@@ -0,0 +1,92 @@
+import React from "react"
+import { FormattedMessage } from "gatsby-plugin-intl"
+import { useStaticQuery, graphql } from "gatsby"
+import Img from "gatsby-image"
+
+import {
+  achievementsImage,
+  imageContainer,
+  andMore,
+  inlineContainer,
+  inlineContainer1,
+} from "./style"
+
+const Achievements = () => {
+  const Image = ({ fluid }) => <Img css={achievementsImage} fluid={fluid} />
```

Comment:
Please put this outside of `<Achievements />`

Reply:
done.